reader-osm/src/test/java/com/graphhopper/GraphHopperIT.java

```diff
@@ -1587,6 +1591,22 @@ private void assertInstruction(Instruction instruction, String expectedName, Str
         assertEquals(expectedPoints, instruction.getPoints().size());
     }
 
+    private void assertTrue(boolean condition) {
```

Comment:
I did not feel like changing all the method calls to the new format, but after adding these methods this could be done with a single click by using the inline refactoring in IntelliJ